src/main/java/com/kadirakinkorkunc/urlshortener/mapper/ConfigurationMapper.java

```diff
@@ -0,0 +1,60 @@
+package com.kadirakinkorkunc.urlshortener.mapper;
+
+import com.kadirakinkorkunc.urlshortener.model.Configuration;
+import com.kadirakinkorkunc.urlshortener.model.ConfigurationRequest;
+import com.kadirakinkorkunc.urlshortener.model.ConfigurationResponse;
+import com.kadirakinkorkunc.urlshortener.util.ConfigurationUtils;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ConfigurationMapper {
+
+    public ConfigurationResponse mapConfigurationToConfigurationResponse(Configuration configuration) {
+        ConfigurationResponse configurationResponse = new ConfigurationResponse();
+        configurationResponse.setKey(configuration.getKey());
+        configurationResponse.setValue(configuration.getValue());
+        return configurationResponse;
+    }
+
+    public List<ConfigurationResponse> mapConfigurationsToConfigurationResponses(List<Configuration> configurationList) {
+        return configurationList.stream().map(this::mapConfigurationToConfigurationResponse).collect(Collectors.toList());
+    }
+
+    public Configuration mapConfigurationReqToConfiguration(ConfigurationRequest configurationRequest) {
+        Configuration configuration = new Configuration();
+        configuration.setKey(configurationRequest.getKey());
+        setConfVal(configuration, configurationRequest.getValue());
+        return configuration;
+    }
+
+    private void setConfVal(Configuration configuration, Object val) {
```

Comment:
Above comment is also includes this
```suggestion
    private void setConfigurationValue(Configuration configuration, Object val) {
```